index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="./style.css" />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Agnes Web Services</title>
+    <link href="https://fonts.googleapis.com/css?family=Cabin|Roboto:400,700|Roboto+Condensed:400,700" rel="stylesheet">
+</head>
+
+<body>
+    <!---Header & navbar -->
+    <header>
+        <nav>
+            <a class="navbar">
+                <a href="#">About us</a>
+                <a href="#">Services</a>
+                <a href="#">Contact</a>
+        </nav>
+    </header>
+
+    <!-- Hero image or video -->
+    <section class="hero">
+        <div class="hero-content">
+            <h1 class="hero-title">
+                We make your website fly
+            </h1>
+            <h2 class="hero-subtitle">
+                Agnes Web Services
+            </h2>
+            <button type="button" class="hero-button">
+                Get started!
+            </button>
+        </div>
+    </section>
+
+    <!--Main section with accordiation and form -->
+
+    <main>
+        <!-- Accordion -->
+        <section class="accordion">
+            <div class="accordion-container">
+                <h3 id="accordion-header-1" class="accordion-header">
+                    <span>Location</span>
```

Comment:
This doesn't need to be wrapped in a `<span>` because now I can't click on your text for your FAQ, just around it. Please check on this!